alert-rules/readme.md

```diff
@@ -0,0 +1,21 @@
+# Alert rules
+This folder contains alert rules for different applications.
+
+The alert rules must be passed to the approriate application.
```

Comment:
What does that mean here?

Reply:
It means that prometheus alert must be sent to prometheus and Loki alerts to Loki